app/dao/notifications_dao.py

```diff
@@ -698,12 +699,29 @@ def dao_old_letters_with_created_status():
     last_processing_deadline = yesterday_bst.replace(hour=17, minute=30, second=0, microsecond=0)
 
     notifications = Notification.query.filter(
-        Notification.updated_at < convert_bst_to_utc(last_processing_deadline),
+        Notification.created_at < convert_bst_to_utc(last_processing_deadline),
```

Comment:
Is this fixing a different bug not mentioned in the PR? Just trying to figure out how it is related

Reply:
We changed this to be `updated_at` to eliminate false positives, for example when a letter is created at 17:29, but doesn't get updated and in the sent bucket until 17:31 (this triggers the alert but the letter ended up in tomorrows bucket).
However, changing the check meant we were not getting alerted when we should. We decided that it's better to get a couple alerts that turn out to be ok then never get the alert.